src/node_crypto.cc

```diff
@@ -5354,6 +5354,25 @@ void GetCurves(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void AddEntropy(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() == 0) {
+    // Delegate entropy generation to OpenSSL, which will add
+    // entropy from system sources
```

Comment:
Is there a preference on actual method name? AddSystemEntropy? We could be very explcit and just call it randPoll, but that probably wouldn't be obvious to the reader what it's actually doing.